wp-content/themes/core/blocks/core/lists/style.pcss

```diff
@@ -0,0 +1,40 @@
+/**
+ * Styles specific to this block
+ */
+
+ul {
+	padding-inline-start: 0;
+}
+
+ol {
+	padding-inline-start: 1.2rem;
+}
+
+li {
```

Comment:
@dpellenwood something odd I noticed while styling lists- when building assets via `npm run dist` I get a [warning from postcss-nesting](http://p.tri.be/i/oQoGgu)

So, something like `ul li { ... }` would throw the warning because it doesn't start w/ a character. Seems odd since we don't have postcss-nesting as a dependency in the project. Have you seen this before? I wondered if something changed w/ the chore updates to dependencies.

Reply:
Dug into this a bit more- requiring nested rules to start w/ a character is part of the spec. So, maybe we want to follow that to stay as close as possible to the spec?

Related: https://chriscoyier.net/2023/03/02/nesting-in-postcss-which-way-to-go/

Reply:
Correct, @jamiepaul . We're following the spec as closely as possible. You can preface any element-selectors with an ampersand. Here's [a bit more clarity](https://webkit.org/blog/13813/try-css-nesting-today-in-safari-technology-preview/) on the current-version of the spec.